app/frontend/packs/nationalities-component.js

```diff
@@ -0,0 +1,118 @@
+const nationalitiesComponent = () => {
+  if (
+    document.querySelector(
+      "[for=candidate-interface-nationalities-form-other-nationality2-field-error]"
+    ) ||
+    document.querySelector(
+      "[for=candidate-interface-nationalities-form-other-nationality3-field-error]"
+    )
+  ) {
+    return;
+  }
+
+  addRemoveLink(
+    "candidate-interface-nationalities-form-other-nationality2-field"
+  );
+
+  addRemoveLink(
+    "candidate-interface-nationalities-form-other-nationality3-field"
+  );
+
+  addAddNationalityButton(
+    "#candidate-interface-nationalities-form-other-other-conditional"
+  );
+
+  hideSection(
+    "candidate-interface-nationalities-form-other-nationality2-field"
+  );
+
+  hideSection(
+    "candidate-interface-nationalities-form-other-nationality3-field"
+  );
+
+  function addRemoveLink(selector) {
+    const labelEl = document.querySelector(`[for=${selector}]`);
+    const removeLink = document.createElement("a");
+    removeLink.innerHTML = "Remove";
```

Comment:
This could a bit better, it could be `Remove <span class="govuk-visually-hidden">{second/third} nationality` depending on what's passed into the `addRemoveLink` function. Would help screen reader users. Up for giving that a spin? ♿

Reply:
Have done 👍